crypto/x509/t_x509.c

```diff
@@ -45,6 +45,12 @@ int X509_print(BIO *bp, X509 *x)
     return X509_print_ex(bp, x, XN_FLAG_COMPAT, X509_FLAG_COMPAT);
 }
 
+/* Prints some string in any case and return error only on I/O error */
+static int ASN1_TIME_print_also_invalid(BIO *bp, const ASN1_TIME *tm)
```

Comment:
Please use something like `asn1_time_print_ignore_invalid`.

Reply:
It turns out that using `ASN1_TIME_check()` is not the right solution.
Its error checking is different to `asn1_time_to_tm()` used by  `ASN1_TIME_print()`.

Changed the solution to introducing a cryptolib-internal function `ASN1_TIME_print_ex()`
with the desired behavior while retaining the original semantics for `ASN1_TIME_print()`.

If this is fine by you I'll extend its use to further internal calls of `ASN1_TIME_print()`.